driver/pace/driver/initialization.py

```diff
@@ -12,14 +12,13 @@
 import pace.fv3core.initialization.analytic_init as analytic_init
 import pace.physics
 import pace.stencils
-import pace.util
-import pace.util.grid
```

Comment:
Again, we still need things like pace.util for things like the quantityfactory so I don't see how this works?